src/ucp/wireup/wireup.c

```diff
@@ -727,6 +727,10 @@ ucp_wireup_send_ep_removed(ucp_worker_h worker, const ucp_wireup_msg_t *msg,
 
 destroy_ep:
     ucp_ep_disconnected(reply_ep, 0);
+    return;
```

Comment:
instead of ep_disconnected+return, can we have ucp_ep_cleanup_lanes() + fall thru to ucp_ep_delete() ?

Reply:
done